tests/test_blend.py

```diff
@@ -29,13 +29,14 @@ def init_data(shape, coords, amplitudes=None, convolve=True, dtype=np.float32):
     if convolve:
         psf_radius = 20
         psf_shape = (2*psf_radius+1, 2*psf_radius+1)
-        psf_center = (psf_radius, psf_radius)
-        target_psf = scarlet.psf.generate_psf_image(scarlet.psf.gaussian, psf_shape, psf_center,
-                                                    amplitude=1, sigma=.9)
+        target_psf = scarlet.psf.generate_psf_image(scarlet.psf.gaussian, psf_shape, sigma=.9)
         target_psf /= target_psf.sum()
 
-        psfs = np.array([scarlet.psf.generate_psf_image(scarlet.psf.gaussian, psf_shape, psf_center,
-                                                        amplitude=1, sigma=1+.2*b) for b in range(B)], dtype=dtype)
+        psfs = np.array([
+            scarlet.psf.generate_psf_image(scarlet.psf.gaussian, psf_shape, sigma=1+.2*b)
+            for b in range(B)
+        ], dtype=dtype)
+        psfs /= psfs.max(axis=(1, 2))[:, None, None]
```

Comment:
What is the max normalisation for here?

Reply:
It's a way to scale the image such that the SED of each source, as detected on the center pixel of each source in `images` is the same as the SED used to initialize the image. So I make use of this particular normalization in all of the tests to ensure that I get out the same SED that I put in.